.goreleaser.yml

```diff
@@ -42,7 +42,19 @@ builds:
       pre: make generate
 
 archives:
-  - format: tar.gz
+  - id: influxctl
+    builds: ["influx"]
+    format: tar.gz
+    wrap_in_directory: true
+    format_overrides:
+      - goos: windows
+        format: zip
+    name_template: 'influxdbctl_{{ .Version }}_{{ .Os }}_{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}'
```

Comment:
The `id` is `influxctl` but the name template is `influxdbctl`. 

Neither of those are the actual name of the `influx` utility, which I think is likely to cause confusion.

Aside from that, this change looks fine.